packages/expo-dev-menu/ios/DevMenuSettings.swift

```diff
@@ -22,7 +22,12 @@ public class DevMenuSettings: NSObject {
       isOnboardingFinishedKey: false,
     ])
 
-    DevMenuMotionInterceptor.isInstalled = DevMenuSettings.motionGestureEnabled
+    /**
+     We don't want to uninstall `DevMemMotionInterceptor`, because otherwise, the app on shake gesture will bring up the dev-menu from the RN.
+     So we added `isEnabled` to disable it, but not uninstall.
```

Comment:
```suggestion
     We don't want to uninstall `DevMenuMotionInterceptor`, because otherwise, the app on shake gesture will bring up the dev-menu from the RN.
     So we added `isEnabled` to disable it, but not uninstall.
```